converter/src/main/java/gov/cms/qpp/conversion/validate/CpcClinicalDocumentValidator.java

```diff
@@ -26,8 +22,8 @@
 	@Override
 	protected void internalValidateSingleNode(Node node) {
 			check(node)
-					.valueIsNotEmpty(MISSING_PRACTICE_SITE_ADDRESS, ClinicalDocumentDecoder.PRACTICE_SITE_ADDR)
-					.singleValue(ONLY_ONE_APM_ALLOWED, ClinicalDocumentDecoder.ENTITY_ID)
-					.childMinimum(ONE_MEASURE_SECTION_REQUIRED, 1, TemplateId.MEASURE_SECTION_V2);
+					.valueIsNotEmpty(ErrorCode.CPC_CLINICAL_DOCUMENT_MISSING_PRACTICE_SITE_ADDRESS, ClinicalDocumentDecoder.PRACTICE_SITE_ADDR)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 128 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)